storage/queue.go

```diff
@@ -170,9 +179,26 @@ func (bq *baseQueue) Add(repl *Replica, priority float64) error {
 // not be added, as the replica with the lowest priority will be
 // dropped.
 func (bq *baseQueue) MaybeAdd(repl *Replica, now proto.Timestamp) {
+	// Load the system config.
+	cfg, err := bq.gossip.GetSystemConfig()
+	if err != nil {
+		log.Error(err)
+		return
+	}
+
+	if !bq.impl.acceptsUnsplitRanges() &&
+		cfg.NeedsSplit(repl.Desc().StartKey, repl.Desc().EndKey) {
+		// Range needs to be split due to zone configs, but queue does
+		// not accept unsplit ranges.
+		if log.V(3) {
+			log.Infof("this replica of %s needs to be split; skipping...", repl)
```

Comment:
this log line is about a range, but the replica will be logged. not a big deal, but reworking the log line would be helpful